services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -86,6 +87,9 @@ class ApplicationSettings(BasicSettings):
     API_SERVER_ALLOWED_HEALTH_CHECK_FAILURES: PositiveInt = 5
     API_SERVER_PROMETHEUS_INSTRUMENTATION_COLLECT_SECONDS: PositiveInt = 5
     API_SERVER_PROFILING: bool = False
+    API_SERVER_TRACING: TracingSettings | None = Field(
+        auto_default_from_env=True, description="settings for opentelemetry tracing"
+    )
 
```

Comment:
so this one is optional this is correct, but the stuff inside TracingSettings should not be optional